app/assets/javascripts/discourse/widgets/hamburger-categories.js.es6

```diff
@@ -5,7 +5,14 @@ import { number } from 'discourse/lib/formatter';
 createWidget('hamburger-category', {
   tagName: 'li.category-link',
 
+  buildAttributes(attrs) {
```

Comment:
mixed feelings on data-category-url vs 

class= 'category-howto-faq'

Seems a bit easier to target the class.